media-libs/libjxl/Manifest

```diff
@@ -0,0 +1 @@
+DIST libjxl-0.7.0_p20220311.tar.gz 17323185 BLAKE2B 53b139f5774bdb335ea32911b4bb388bdf012ce0e787b1a6b57e8c4a47b19a81bfa85dccebccb034e8241057b2d98e1b2784dfa5555abba0ffbd558a2765d8fc SHA512 ffaf8c505035991c2ca2340109597c870b1599fd884f426763d00f5e6033230e8b889d6985b7054338cc1d3e13a77a92078c655833152ff0fc1ed03d30486e68
```

Comment:
I think the more correct version would be `0.6.1_p...`

Reply:
When you look in `/usr/lib64/pkgconfig/libjxl.pc` there is `Version: 0.7.0`
The API is already different than in 0.6.1

Reply:
What about 0.7.0_preDate. If they release 0.7.0 then the _p would make it downgrade, but _pre is lower than .0.

Reply:
OK, `_pre` is good. I'll change that. BTW according the info I have, the next released version will be 1.0. But who knows...